packages/lib/test/builder.ts

```diff
@@ -176,7 +179,7 @@ export const buildCalendarEvent = (event?: Partial<CalendarEvent>): CalendarEven
     customInputs: {},
     additionalNotes: faker.lorem.paragraph(),
     organizer: buildPerson(),
-    videoCallData: buildVideoCallData(),
+    ...(!omitVideoCallData && { videoCallData: buildVideoCallData() }),
```

Comment:
Just curious. Why is this needed?

Reply:
When generating the ics string we look for video call data first and then the location. Since video call data was always included in the event builder the ics string would always fallback to the video call url.